### PR TITLE
JE-316: Upgraded bundles

### PR DESCRIPTION
**Pull request by tuj**

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -7,10 +7,10 @@ jobs:
         steps:
             -   uses: actions/checkout@master
             -   name: Setup PHP, with composer and extensions
-                uses: shivammathur/setup-php@master
+                uses: shivammathur/setup-php@v2
                 with:
                     php-version: 7.2
-                    extension-csv: ctype, dom, iconv, json
+                    extension: ctype, dom, iconv, json
```
Comment (rimi-itk) [inline .github/workflows/pr.yml]: This should be `extensions` (cf. https://github.com/shivammathur/setup-php#heavy_plus_sign-php-extension-support)

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -22,10 +22,10 @@ jobs:
         steps:
             - uses: actions/checkout@master
             - name: Setup PHP, with composer and extensions
-              uses: shivammathur/setup-php@master
+              uses: shivammathur/setup-php@v2
               with:
                   php-version: 7.2
-                  extension-csv: ctype, dom, iconv, json
+                  extension: ctype, dom, iconv, json
```
Comment (rimi-itk) [inline .github/workflows/pr.yml]: This should be `extensions` (cf. https://github.com/shivammathur/setup-php#heavy_plus_sign-php-extension-support)

File: .github/workflows/pr.yml [hunk 3]
```diff
@@ -45,10 +45,10 @@ jobs:
...
```
Comment (rimi-itk) [inline .github/workflows/pr.yml]: You've got the idea …

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Minor changes in GitHub Actions needed.

File: .github/workflows/pr.yml (hunk 1 above)
Comment (tuj) [reply .github/workflows/pr.yml]: Right you are... weird that it did not cause an error

Comment (rimi-itk) [reply .github/workflows/pr.yml]: I'm also lucky – I read https://github.com/shivammathur/setup-php/issues/440#issuecomment-811939621 just yesterday.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**